stonesoup/models/transition/base.py

```diff
@@ -7,6 +7,7 @@
 
 from ..base import Model, GaussianModel
 from ...base import Property
+from ...types.array import StateVectors
 from ...types.state import StateVector
```

Comment:
Odd, `StateVector` should have been from `array` module as well. May as well fix that in this PR.
```suggestion
from ...types.array import StateVector, StateVectors
```